dev-utils/visualise-ui-hierarchy/Makefile

```diff
@@ -0,0 +1,46 @@
+CWD := $(shell pwd)
+POLYPOD_FEATURES_PATH := $(CWD)/../../features
+RCH := node $(CWD)/react-component-hierarchy/bin/rch.js -e
```

Comment:
You can probably use `npx` here to get the path right.

Reply:
That'd require people to globally install `npx`, right? It can be called from scripts defined in `package.json` because NPM has it in the path then, but this is a Makefile, so it'd be executed with the regular shell.

Probably a good idea not to overthink this code too much - it's not production relevant.